tests/unit/test_ps_model.py

```diff
@@ -550,3 +549,22 @@ def test_fill_low_altitude_with_zero_wind(
     # if air temperature is computed, the model can estimate fuel flow
     assert np.sum(np.isfinite(fl1["fuel_flow"])) == 499  # all but the last value
     assert np.sum(np.isfinite(fl2["fuel_flow"])) == 242  # missing everything below lowest met level
+
+
+def test_ps_flight_on_fleet() -> None:
+    """Confirm the PSFlight model can be evaluated on a Fleet object.
+
+    This is simply a smoke test -- no data is actually checked.
+    """
+    df_flight = pd.read_csv(get_static_path("flight.csv"))
+    assert len(df_flight) == 254
+
+    fl1 = Flight(df_flight.iloc[::2], aircraft_type="A320", flight_id=1)
+    fl2 = Flight(df_flight.iloc[1::2], aircraft_type="A333", flight_id=2)
+    fleet = Fleet.from_seq([fl1, fl2])
+
+    fleet["air_temperature"] = fleet.T_isa()
+
+    ps_model = ps.PSFlight(fill_low_altitude_with_zero_wind=True)
+    out = ps_model.eval(fleet)
```

Comment:
This is a more generic `Fleet` question:

I had a question here recently about how `Fleet` objects handle attrs. Would we be able to have a test here the shows how the attrs show up in "out"?

I would expect to find `aircraft_type` somewhere in the output.

```python
assert "flight_id" in out.data and "aircraft_type" in out.data
```

or is there another place these attrs get stored?

Reply:
There's not a great way to handle attrs on `Fleet` instances. The current paradigm is to include global properties in `Fleet.attrs` and individual flight properties in `Fleet.fl_attrs`. I expanded the test a bit to check these out: 5602a03ac229417d3f88c517b7ce55ddc83e4522

We haven't made consistent use of `Fleet.attrs` throughout pycontrails. I think the best practice would be to check for a scalar quantity in `Fleet.attrs` first, then to look within `Fleet.fl_attrs` next ... you can see how this gets complicated tricky.

Reply:
oh great! this answers my question. Is there any reason not to use `Fleet.attrs` instead of `Fleet.fl_attrs`? Was it a compatibility or serialization question?

Reply:
We definitely could have used `Fleet.attrs` instead. We always end up creating a dictionary of the form `flight_id: fl.attrs` for each `fl in fleet`, and we make use of it in various `Fleet` methods, so I attached it as an explicit property on the instance rather than assuming `fleet.attrs` included separate keys for each flight_id.

Reply:
Okay -- so you're suggesting we overload `Fleet.attrs` to hold `{flight_id: flight_properties}`. I think could be a reasonable way to proceed. I think I avoided this in the past because `attrs["crs"]` was getting in my way.

I think I'll hold off for now, but may do this whenever I iterate again on `Fleet`.